api/files/serializers.py

```diff
@@ -307,10 +316,21 @@ def get_absolute_url(self, obj):
 
 
 class FileSerializer(BaseFileSerializer):
-    node = RelationshipField(related_view='nodes:node-detail',
-                             related_view_kwargs={'node_id': '<node._id>'},
-                             help_text='The project that this file belongs to'
-                             )
+    node = ShowIfVersion(
+        FileNodeRelationshipField(
+            related_view='nodes:node-detail',
+            related_view_kwargs={'node_id': '<target._id>'},
+            help_text='The project that this file belongs to'
+        ),
+        min_version='2.0', max_version='2.6'
```

Comment:
Latest version as of this writing is 2.7, so `max_version` will need to be updated.